GTG/gtk/editor/editor.py

```diff
@@ -179,40 +168,7 @@ def __init__(self, app, task):
         textview_focus_controller.connect("leave", self.on_textview_focus_out)
         self.textview.add_controller(textview_focus_controller)
 
-        tags = task.tags
-        text = self.task.content
-        title = self.task.title
-
-        # Insert text and tags as a non_undoable action, otherwise
```

Comment:
This block of code has moved to the TaskView class. The logic hasn't changed besides the way the objects are named (`self.insert()` instead of `self.textview.insert()` etc) and the fact that the TextView calls `is_new()` on the task object.